Source/DafnyTestGeneration/README.md

```diff
@@ -0,0 +1,123 @@
+# Test Generation for Dafny
+
+[**Purpose**](#purpose) <br>
+[**General Approach**](#general-approach) <br>
+[**Implementation**](#implementation) <br>
+[**How to Generate Tests?**](#how-to-generate-tests) <br>
+[**How to Run Tests?**](#how-to-run-tests)
+
+## Purpose
+
+Block- and path-coverage tests of Dafny code can be used to:
```

Comment:
Could you elaborate more on these use-case? For example, how would I identify dead-code with this feature?